src/main/java/nl/altindag/log/LogCaptor.java

```diff
@@ -29,12 +29,7 @@
 import java.time.Instant;
 import java.time.ZoneOffset;
 import java.time.ZonedDateTime;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
+import java.util.*;
```

Comment:
I prefer to explicitly list all the used objects instead of the wildcard. It just just a personal preference... Would it be possible for you to revert this change?

Reply:
Done